Formula/vlang.rb

```diff
@@ -12,8 +12,8 @@ class Vlang < Formula
   end
 
   resource "vc" do
-    url "https://github.com/vlang/vc/archive/0884d70.tar.gz"
-    sha256 "6fc0eab593ef8508cbeaadc79a86dc90e3de2ec4646236e39722930a2484eebe"
+    url "https://github.com/vlang/vc/archive/0e58ced.tar.gz"
```

Comment:
Could you put a comment here regarding how you got this commit for `vc`? It would be helpful for future maintainers/contributors when bumping this formula down the road.

Reply:
@alebcay Good idea. Took me some while to figure this out myself as well. Are you ok with the wording?

Reply:
I personally think it looks all right - I was able to follow it and figure out which commit.

Reply:
> @alebcay Good idea. Took me some while to figure this out myself as well. Are you ok with the wording?

I'm sorry 😄 Still, feel free to ask next time 😉